tools/schemacode/bidsschematools/data/tests/test_rules.py

```diff
@@ -81,9 +82,21 @@ def test_rule_objects(schema_obj):
                     object_values = [
                         value["value"] for value in schema_obj["objects"][object_type].values()
                     ]
-                else:
+                elif object_type in [
+                    "columns",
+                    "common_principles",
+                    "datatypes",
+                    "entities",
+                    "enums",
+                    "files",
+                    "formats",
+                    "metadata",
+                    "modalities",
+                ]:
                     # But other object types are referenced via their keys
                     object_values = list(schema_obj["objects"][object_type].keys())
+                else:
+                    raise AssertionError(f"Object type {object_type} not implemented.")
```

Comment:
This makes more sense to me to put up right under `for object_type in schema_obj.objects:` rather than checking repeatedly.